designs/umbra/src/front.mjs

```diff
@@ -0,0 +1,232 @@
+import { base } from './base.mjs'
+import { units } from '@freesewing/core'
```

Comment:
Not sure why you're using `units` from core here, rather than deconstruct like you did in base.

The difference is that the deconstructured `units` is aware of the units requested by the user, and will format accordingly. The `units` from core does not know, and so you need to tell it what units to use (or use the default, which is metric).

Check how `units` is loaded in `base.mjs`, and you should do the same here, unless there's some specific reason that I'm missing.

Reply:
> Not sure why you're using units from core here, rather than deconstruct like you did in base.

Probably because I applied a quickfix from the IDE instead of looking at the proper way.